### DISCUSSION TITLE
En passant checkmate

### DISCUSSION THREAD

**Discussion by akaKarthick** [czbar/ChessForge#637] [General]
when the checkmate was achieved through the en passant capture, the notation is sometimes not ending with # checkmate symbol. 
i dont know more about en passant. so i could be wrong. 

And importantly the plus symbol is changing itself into # after couple of minutes. ( so is this lag right?)

https://youtu.be/j5BKfO80SKM

https://drive.google.com/file/d/1kwd03oa6x3IcArESYBal9Jdj5Og2q4Jw/view?usp=sharing

**Comment by czbar** [comment]
I keep missing the "discussion" items :(   

The checkmate detection was incorrect in some circumstance  (to do with pawn moving from their initial row rather than en passant).  Will be fixed in the next private build (1.2.2).
